app/controllers/application_controller.rb

```diff
@@ -48,7 +48,15 @@ def peek_enabled?
   private
 
   def current_user
-    @current_user ||= User.find_or_create_by(email_address: session[:user]['email']) if session[:user] && session[:user]['email']
+    @current_user ||= find_current_user
+  end
+
+  def find_current_user
+    if ENV['FORCE_USER_ID']
```

Comment:
Can we at least add a check for `!Rails.env.production?`. Ok to merge this in otherwise until we think of a better solution.